src/games/RomUtils.cpp

```diff
@@ -30,6 +30,12 @@ int readRam(const System* system, int offset) {
 }
 
 
+/* writes a byte at a memory location between 0 and 128 */
+void writeRam(System* system, int offset, unsigned char value) {
+
```

Comment:
No whitespace here (or be consistent across functions).

Reply:
I'm doing my best, but since it is different from what he had before, sometimes things something slips in. Thanks for pointing that out.